examples/profile-client/src/main/java/com/palantir/example/profile/schema/generated/UserProfileTable.java

```diff
@@ -1337,9 +1337,9 @@ public int hashCode() {
             }
 
             @Override
-            public int compareTo(CookiesIdxRow cookiesIdxRow) {
```

Comment:
No, there wasn't, I just regenerated the classes. The change is due to a previous PR where some linting was applied that changed the param name in the generated code.